### ISSUE TITLE
loot@ too big images

### ISSUE THREAD

**Issue by kito90** [Mytherin/Tibialyzer#106]
Hey, how to re-size the creature list shown at loot@ and also how to remove unnecesary stuff like exp/hr, loot, profit, waste... since I am seeing them all at summary.

Also how to remove the gold coins from the profit earned... I am ms hunting at banuta and I dont have cap to loot gps xD

**Comment by kito90** [comment]
This is because the loot screen start to grow up and then it pass over the cooldown icons and also over the chat.

**Comment by KuerbisGnu** [comment]
Hello,
take a look at the wiki
https://github.com/Mytherin/Tibialyzer/wiki/Settings-Guide#database-tab
There you can find all information how to remove gold coins and other items from your counted loot.

You can't remove anything in the loot@ notification.
The only way to resize it is this:
![image](https://cloud.githubusercontent.com/assets/17644820/16259278/1fc5a9b2-3862-11e6-8db2-2ec400ca1bef.png)
Just leftclick "-" or "+" with your mouse.

**Comment by kito90** [comment]
I pressed it but keep growing up and growing up.... Having where the "-" let me has it the smallest size on the screen. But keep growing till it covers the coold down skils and the chat... I have a 1920x900 screen and is not enought. The size of the monsters is too big... There should be an option to resize them.

**Comment by KuerbisGnu** [comment]
Yeah it grows in y-axis but not x-axis. But you can see near all the information in summary@.
Maybe you can send a screenshot of your problem because I can't figure out exactly what is the problem. Maybe you just need to change the position or something else.

**Comment by kito90** [comment]
Using summary@ right now, since loot@ was growing up so much that goes up the chat and cooldown bar.

**Comment by kito90** [comment]
Ok, check this Image to check this.... It has the lower size it could be.

http://i.imgur.com/gu4486O.jpg

**Comment by KuerbisGnu** [comment]
Take a look at this:
![image](https://cloud.githubusercontent.com/assets/17644820/16712558/ef83531c-468a-11e6-9bc9-dae8d5fd4bbb.png)
Play around with these options. It should help you I guess.
